.travis.yml

```diff
@@ -40,27 +40,27 @@ matrix:
         - DESTINATION="OS=9.1,name=iPhone 6s" SDK=iphonesimulator9.3 TYPE="UNIT"
     - env:
         - DESTINATION="OS=9.3,name=iPhone 6s Plus" SDK=iphonesimulator9.3 TYPE="FUNCTIONAL"
-    - osx_image: xcode8.2
+    - osx_image: xcode9.2
       env:
-        - DESTINATION="OS=10.2,name=iPhone 6s" SDK=iphonesimulator10.2 TYPE="CARTHAGE"
-    - osx_image: xcode8.2
+        - DESTINATION="OS=10.1,name=iPhone 6s" SDK=iphonesimulator11.2 TYPE="CARTHAGE"
```

Comment:
Maybe one target running on 10.2?

Reply:
we do right:

- DESTINATION="OS=10.2,name=iPhone 7 Plus" SDK=iphonesimulator11.2 TYPE="EXAMPLE_PODS"